src/main/java/org/oasis_eu/portal/ui/UIApp.java

```diff
@@ -1,10 +1,10 @@
-package org.oasis_eu.portal.model.dashboard;
+package org.oasis_eu.portal.ui;
 
 /**
  * User: schambon
  * Date: 11/19/14
  */
-public class DashboardApp {
+public class UIApp {
```

Comment:
not really in sync with this renaming. we lose the fact that it is a representation of an app for the dashboard.